timemachine/fe/free_energy.py

```diff
@@ -37,6 +38,16 @@ class MDParams:
     n_frames: int
     n_eq_steps: int
     steps_per_frame: int
+    local_steps: int = 0
+    k: float = 1_000.0
+    min_radius: float = 1.0
+    max_radius: float = 3.0
+    freeze_reference: bool = True
+
+    def __post_init__(self):
+        assert 0.1 <= self.min_radius <= self.max_radius
+        assert self.local_steps <= self.steps_per_frame
```

Comment:
Should assert positive?

Reply:
Asserted in https://github.com/proteneer/timemachine/pull/1026/commits/9a63ea2b3967cf830b6b599e06dfeddf96b69a45